src/domainManagementUI/src/app/services/tab-services/domain-details-tabs.service.ts

```diff
@@ -147,15 +147,22 @@ export class DomainDetailsTabService {
     });
   }
 
-  _buildAttributesForm() {
+  _buildAttributesForm(name: string) {
     this.attributes_form = new FormGroup({});
-
     this.templateSvc
       .getTemplateAttributes()
-      .subscribe((attributes: string[]) => {
-        attributes.forEach((att: string) => {
+      .subscribe((attributes: { [key: string]: string }) => {
+        Object.entries(attributes).forEach((key, value) => {
           const attribute = new TemplateAttribute();
-          attribute.key = att;
+
+          if (key[0] == 'name') {
+            key[1] = name;
+          } else if (key[0] == 'email') {
+            key[1] = `info@${name}`;
+          }
```

Comment:
Oh, I see why you're not having the backend generate an email now. However, why do you pass domain name into name? Is that supposed to be a domain name or a name of a person?

Reply:
It's supposed to be the name of the website, not a person, which usually correlates with the domain name. I figured we can pre-populate it with the domain name and leave that up to the user to change it.